File: AFMobile/AFAndroid/app/src/main/java/cz/cvut/fel/matyapav/afandroid/enums/LayoutOrientation.java
package cz.cvut.fel.matyapav.afandroid.enums;

/**
 * Enum of supported layout orientations for form
 *
 * @author Pavel Matyáš (matyapav@fel.cvut.cz).
 * @since 1.0.0..
 */
public enum LayoutOrientation {

    AXISX("AXISX"), AXISY("AXISY");

    private String orientation;

    LayoutOrientation(String orientation) {
        this.orientation = orientation;
    }

    public String getName() {
        return orientation;
    }


}

File: AFMobile/AFWinPhone/AFWinPhone/enums/LayoutOrientation.cs
using System;

namespace AFWinPhone.enums
{
    public sealed class LayoutOrientation
    {
        public static readonly LayoutOrientation AXISX = new LayoutOrientation("AXISX");
        public static readonly LayoutOrientation AXISY = new LayoutOrientation("AXISY");

        private String orientation;

        LayoutOrientation(String orientation)
        {
            this.orientation = orientation;
        }


        public String getName()
        {
            return this.orientation;
        }
    }
}
